Documentation/network/concepts/ipam/multi-pool.rst

```diff
@@ -102,6 +102,25 @@ described above.
               - 10.20.0.0/16
             maskSize: 24
 
+Updating existing CiliumPodIPPools
+----------------------------------
+
+Once you configure the ``CiliumPodIPPools``, you cannot update the existing pool. For example, 
+you can't change the default pool to a different CIDR or add an IPv6 CIDR to the default pool. 
+This restriction prevents pods from receiving IPs from a new range while some pods still use 
+the old IP pool on the same nodes. If you need to update the existing CiliumPodIPPools, Please
+follow these steps. 
+
+1. Update your existing pool through ``autoCreateCiliumPodIPPools`` in helm values.
+2. Delete the existing ``CiliumPodIPPools`` from CR and restart the Cilium operator to create new ``CiliumPodIPPools``.
+3. Cordon the nodes (Node Group 1) where you want to update the CIDR to the new pool, and evict pods 
```

Comment:
I suggest to clarify what is meant with "Node Group 1" and "Node Group 2" before the steps list. Maybe we can add a small section ahead stating something like "Suppose that we are in this scenario ... and we want to ...".